runtime/runtime/src/state_viewer/mod.rs

```diff
@@ -526,6 +538,8 @@ mod tests {
             cache: None,
             #[cfg(feature = "protocol_feature_evm")]
             evm_chain_id: 0x99,
+            #[cfg(feature = "protocol_feature_block_hash_host_fn")]
+            block_hash_provider: Arc::new(near_primitives::types::NullBlockHashProvider),
```

Comment:
Unrelated to the PR, but it seems like these tests would benefit from refactoring the construction of the ViewApplyState to a sligle place. It feels like a fairly frequently changing thing, and updating all the call sites is tedious.